docs/custom-flows/email-password.mdx

```diff
@@ -387,6 +387,146 @@ To sign up a user using their email and password, you must:
 
     </InjectKeys>
   </Tab>
+  <Tab>
+    First create a route group with a layout that redirects users if they're already signed in, as shown in the following example:
+
+    ```tsx filename="/app/(auth)/_layout.tsx"
+    import { Redirect, Stack } from "expo-router";
+    import { useAuth } from "@clerk/clerk-expo";
+
+    export default function GuestLayout() {
+      const { isSignedIn } = useAuth();
+
+      if (isSignedIn) {
+        return <Redirect href={"/dashboard"} />;
+      }
+
+      return <Stack />;
+    }
+    ```
+
+    Then create a sign-up page that sends users an email verification code to create an account.
+
+    ```tsx filename="/app/(auth)/sign-up.tsx"
+
+    import * as React from "react";
+    import { Text, TextInput, TouchableOpacity, View } from "react-native";
+    import { useSignUp } from "@clerk/clerk-expo";
+
+    export default function SignUpScreen() {
+      const { isLoaded, signUp, setActive } = useSignUp();
+
+      const [firstName, setFirstName] = React.useState("");
+      const [lastName, setLastName] = React.useState("");
+      const [emailAddress, setEmailAddress] = React.useState("");
+      const [password, setPassword] = React.useState("");
+      const [pendingVerification, setPendingVerification] = React.useState(false);
+      const [code, setCode] = React.useState("");
+
+      // Start the sign up process.
+      const onSignUpPress = async () => {
+        if (!isLoaded) {
+          return;
+        }
+
+        try {
+          await signUp.create({
+            firstName,
+            lastName,
+            emailAddress,
+            password,
+          });
+
+          // send the email.
+          await signUp.prepareEmailAddressVerification({ strategy: "email_code" });
+
+          // change the UI to our pending section.
+          setPendingVerification(true);
+        } catch (err: any) {
+          console.error(JSON.stringify(err, null, 2));
+        }
+      };
+
+      // Try to verify the user with the provided email code.
+      const onPressVerify = async () => {
+        if (!isLoaded) {
+          return;
+        }
+
+        try {
+          const completeSignUp = await signUp.attemptEmailAddressVerification({
+            code,
+          });
+
+          await setActive({ session: completeSignUp.createdSessionId });
+        } catch (err: any) {
+          console.error(JSON.stringify(err, null, 2));
+        }
+      };
+
+      return (
+        <View>
+          {!pendingVerification && (
```

Comment:
Not necessary, but we have these in the next.js example so you can decide if we want these 🤷‍♀️ 

```suggestion
          // Display the initial sign-up form to capture the email and password
          {!pendingVerification && (
```